src/styles/CHANGELOG.md

```diff
@@ -9,10 +9,13 @@ This repo is in Prerelease. When it is released, this project will adhere to [Se
 ========
 
 ## [0.0.23]
-- added `Template` styles
+
+-   added `Template` styles
```

Comment:
Do we want prettier to ignore .md files?

Reply:
See .scss comment